### PR TITLE
Add 1az Config for default region

### PR DESCRIPTION
**Pull request by prsurve**
Signed-off-by: prsurve <prsurve@redhat.com>

### CODE REVIEW

File: conf/ocsci/aws_ipi_1az.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (petr-balogh) [inline conf/ocsci/aws_ipi_1az.yaml]: I know it's default value for our deployment but when you are explicit in the name of the IPI deployment type. What about to add it also here:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/production-aws-ipi.yaml#L6

Or second option to not base it on deployment type, rename it to aws_1az.yaml and for UPI will will need to align to use the same if possible. I prefer second option.

Comment (prsurve) [reply conf/ocsci/aws_ipi_1az.yaml]: will check with UPI deployment as well and update file name

Comment (petr-balogh) [reply conf/ocsci/aws_ipi_1az.yaml]: BTW if the UPI deployment won't support it, we will need to open issue for it and you can just put comment to this config file at the start of this file that UPI is not supported and link to the issue.

Comment (petr-balogh) [inline conf/ocsci/aws_ipi_1az.yaml]: Also please put some comment at the start of the config file like:
`# This config is for setup cluster in 1 Availability Zone for AWS`

Thanks

Comment (prsurve) [reply conf/ocsci/aws_ipi_1az.yaml]: - Addressed Comments
- Updated yaml with issue url

Comment (prsurve) [reply conf/ocsci/aws_ipi_1az.yaml]: Done

Comment (vasukulkarni) [reply conf/ocsci/aws_ipi_1az.yaml]: if this is default for aws-ipi, why not just add it to existing aws-ipi file?

Comment (prsurve) [reply conf/ocsci/aws_ipi_1az.yaml]: @vasukulkarni this config can be also used for aws-upi so removed aws-ipi file, currently aws-upi installation with this config is not working so i created an issue https://github.com/red-hat-storage/ocs-ci/issues/991

**Pull request merged**